src/components/FoodBox.jsx

```diff
@@ -0,0 +1,17 @@
+import { Card, Col, Button } from 'antd';
+
+export const FoodBox = ({ food }) => {
+  return (
+    <Col>
+      <Card title={food.name} style={{ width: 230, height: 300, margin: 10 }}>
+        <img src={food.image} height={60} alt="food" />
+        <p>Calories: {food.calories}</p>
+        <p>Servings: {food.servings}</p>
+        <p>
+          <b>Total Calories: {food.calories * food.servings} </b> kcal
```

Comment:
specifically here, for a newly created food this does assume this is a number. Now you are lucky, since JS handles this somewhat gracefully for you. 
i.e. 
"10" * 5 would return 50. 
5 * "10" would return 50. 

However if you do, 
"10" + 1 
you'd be in trouble. 
![image](https://user-images.githubusercontent.com/25333032/201481916-4d723e04-3df1-4c90-8dae-3a96378d3244.png)